front/src/components/filters/FilterSection.tsx

```diff
@@ -1,28 +1,27 @@
 import React from "react";
-import { FilterType } from "../../schema/FilterSchema";
-import { useGetFilters } from "../../services/filterService";
-import { useStyles } from "./FilterSectionStyles";
+import {FilterType} from "../../schema/FilterSchema";
+import {useGetFilters} from "../../services/filterService";
+import {useStyles} from "./FilterSectionStyles";
 import RangeFilter from "./range-filter/RangeFilter";
 import CheckboxFilter from "./checkbox-filter/CheckboxFilter";
-import { GCAccordion } from "../../gc-components/accordion/GCAccordion";
+import {GCAccordion} from "../../gc-components/accordion/GCAccordion";
 
 const FilterSection = () => {
+    const {classes} = useStyles();
+
     const [data, isLoading, isError, isSuccess] = useGetFilters();
-    const { classes } = useStyles();
+
     if (isSuccess) {
         return (
             <div className={classes.section}>
                 {data.map((filterItem) => (
                     <GCAccordion
+                        key={filterItem.filterKey}
                         className={classes.filter}
                         showName={filterItem.showName}
-                        panel={
-                            filterItem.filterType == FilterType.RANGE ? (
-                                <RangeFilter filter={filterItem} />
-                            ) : (
-                                <CheckboxFilter filter={filterItem} />
-                            )
-                        }
+                        panel={filterItem.filterType == FilterType.RANGE
```

Comment:
Please avoid using double equals, always use triple one, so it leaves less room for error